docs/pages/framework/world/index.md

```diff
@@ -3,9 +3,9 @@
 The World contract acts as a central database that stores all models and systems while managing permissions. Think of it as a global store for your application's state.
 
 :::tip[Resource]
-In the Dojo world, we define the term `resource` to refer to any model or system that can be registered in the world. The world itself is a special resource, with the value `0`.
+In the Dojo world, we define the term `resource` to refer to any model or system that can be registered in the world. The world itself is a special resource, with the resource ID `0`.
```

Comment:
```suggestion
In the Dojo world, we define the term `resource` to refer to any model or system that can be registered in the world. The world itself is a special resource, with the resource selector `0`.
```
Generally speaking I think we should use `dojo selector` or `selector` instead of ID, which is also something we use with `entity_id`. wdyt?

Reply:
Yep, you're right 👍